### PR TITLE
Displays syntax errors in a better way

### PR DESCRIPTION
**Pull request by NotMyFault**
## Description
FAVS swallows command syntax errors and responds with a generic wrong syntax message. This gets difficult to figure out, if you're unfamiliar with each command's syntax. The change proposed mitigates this scenario by pointing out the wrong syntax.

## Checklist

- [X] Make sure you are opening from a topic branch (**/feature/fix/docs/ branch** (right side)) and not your main branch.
- [X] Ensure that the pull request title represents the desired changelog entry.
- [X] I tested my changes and approved their functionality.
- [X] I ensured my changes do not break other parts of the code
- [X] I read and followed the [contribution guidelines](https://github.com/IntellectualSites/.github/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: src/main/java/com/thevoxelbox/voxelsniper/brush/type/ErodeBrush.java [hunk 1]
```diff
@@ -70,7 +70,7 @@ public void handleCommand(String[] parameters, Snipe snipe) {
                     messenger.sendMessage(ChatColor.LIGHT_PURPLE + "Brush preset set to: " + preset.getName());
                     return;
                 } catch (IllegalArgumentException exception) {
-                    messenger.sendMessage(ChatColor.RED + "Invalid preset.");
+                    messenger.sendMessage(ChatColor.RED + "Invalid preset: " + preset.getName());
```
Comment (Aurelien30000) [inline src/main/java/com/thevoxelbox/voxelsniper/brush/type/ErodeBrush.java:73]: ~~Preset is invalid, getting its name is not a good idea imo.~~

File: src/main/java/com/thevoxelbox/voxelsniper/brush/type/GenerateTreeBrush.java [hunk 2]
```diff
@@ -95,8 +95,8 @@ public void handleCommand(String[] parameters, Snipe snipe) {
         if (firstParameter.equalsIgnoreCase("info")) {
             messenger.sendMessage(ChatColor.GOLD + "This brush takes the following parameters:");
             messenger.sendMessage(ChatColor.AQUA + "/b gt default -- Restores default params.");
-            messenger.sendMessage(ChatColor.AQUA + "/b gt lt [t] -- Sets leaf type to t. (e.g. oak)");
-            messenger.sendMessage(ChatColor.AQUA + "/b gt wt [t] -- Sets wood type to t. (e.g. oak)");
+            messenger.sendMessage(ChatColor.AQUA + "/b gt lt [t] -- Sets leaf type to t. (e.g. oak_leaves)");
+            messenger.sendMessage(ChatColor.AQUA + "/b gt wt [t] -- Sets wood type to t. (e.g. oak_leaves)");
```
Comment (Aurelien30000) [inline src/main/java/com/thevoxelbox/voxelsniper/brush/type/GenerateTreeBrush.java]: Wood type is still oak and not oak_leaves ;)

File: src/main/java/com/thevoxelbox/voxelsniper/brush/type/rotation/Rotation2DBrush.java [hunk 3]
```diff
@@ -39,7 +39,7 @@ public void handleCommand(String[] parameters, Snipe snipe) {
                     messenger.sendMessage(ChatColor.GREEN + "Angle set to: " + DECIMAL_FORMAT.format(this.angle) + " gradians " +
                             "(" + degreesAngle + " degrees)");
                 } else {
-                    messenger.sendMessage(ChatColor.RED + "Invalid number.");
+                    messenger.sendMessage(ChatColor.RED + "Invalid number: " + parameters[0]);
```
Comment (Aurelien30000) [inline src/main/java/com/thevoxelbox/voxelsniper/brush/type/rotation/Rotation2DBrush.java]: Use firstParameter intialized above (you should also replace it at line 36).

File: src/main/java/com/thevoxelbox/voxelsniper/brush/type/rotation/Rotation2DVerticalBrush.java [hunk 4]
```diff
@@ -41,7 +41,7 @@ public void handleCommand(String[] parameters, Snipe snipe) {
                     messenger.sendMessage(ChatColor.GREEN + "Angle set to: " + DECIMAL_FORMAT.format(this.angle) + " gradians " +
                             "(" + degreesAngle + " degrees)");
                 } else {
-                    messenger.sendMessage(ChatColor.RED + "Invalid number.");
+                    messenger.sendMessage(ChatColor.RED + "Invalid number: " + parameters[0]);
```
Comment (Aurelien30000) [inline src/main/java/com/thevoxelbox/voxelsniper/brush/type/rotation/Rotation2DVerticalBrush.java]: Same, firstParameter.

**Review by Aurelien30000 — CHANGES_REQUESTED** [review]
Of course, interesting changes, ty!

File: src/main/java/com/thevoxelbox/voxelsniper/brush/type/ErodeBrush.java (hunk 1 above)
Comment (Aurelien30000) [reply src/main/java/com/thevoxelbox/voxelsniper/brush/type/ErodeBrush.java:73]: Actually, I’m wondering if the IllegalArgumentException catch is necessary 🤔

File: src/main/java/com/thevoxelbox/voxelsniper/brush/type/GenerateTreeBrush.java (hunk 2 above)
Comment (NotMyFault) [reply src/main/java/com/thevoxelbox/voxelsniper/brush/type/GenerateTreeBrush.java]: That'd be `oak_log` then.

**Review by Aurelien30000 — APPROVED** [review]

**Pull request merged**